base/linalg/tridiag.jl

```diff
@@ -14,6 +14,13 @@ immutable SymTridiagonal{T} <: AbstractMatrix{T}
     end
 end
 
+"""
+    SymTridiagonal(dv, ev)
+
+Construct a symmetric tridiagonal matrix from the diagonal and first diagonal,
+respectively. The result is of type `SymTridiagonal` and provides efficient specialized
+eigensolvers, but may be converted into a regular matrix with [`full`](:func:`full`). The length of `ev` must be one less than the length of `dv`.
```

Comment:
Then why do we have it the other way for `Bidiagonal`? I could see that really messing with someone.